### PR TITLE
Update contributing guidelines with steps to update API docs 

### LINKED ISSUE #87: Document the preferred way to merge PRs in the contributing guidelines.
If an answer exists, it should be documented.

### PR DESCRIPTION
**Pull request by ashfall**
Also documents the preferred way to merge a PR on Github (Fixes #87).

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ Contributing guidelines
 * Ensure your code is tested and documented. Patches without 100% test coverage
   or documentation will not be accepted.
 * Write `good commit messages`_.
+* Once you are ready with your changes and before you submit it for review to
+  make sure the API docs are updated with your changes, please run::
+
+    sphinx-apidoc -f -o docs/_autodoc tls/ --private
```
Comment (sigmavirus24) [inline CONTRIBUTING.rst]: What if we use a tox testenv to provide this to the user so they can do

```
tox -e gendoc
```

And that way they don't have to worry about installing sphinx in a virtualenv or however else they might do it.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -47,6 +51,17 @@ for committers:
   merged until this is rectified.
 * All merged patches must have 100% test coverage.
 
+Reviewing Guidelines
+------------------------
+
+* Make sure the API docs are updated with the changes. If they are not, request
+  the contributor run `sphinx-apidoc` to update them.
```
Comment (sigmavirus24) [inline CONTRIBUTING.rst]: Friendly reminder that reStructuredText uses double back-ticks to pre-format the text, e.g.,

```rst
the contributor run ``sphinx-apidoc`` to update them.
```

Comment (ashfall) [inline CONTRIBUTING.rst]: Fixed, thanks! :)

File: CONTRIBUTING.rst (hunk 1 above)
Comment (ashfall) [inline CONTRIBUTING.rst]: Good idea. Looking into it.

**Comment by sigmavirus24** [comment]
This looks excellent to me. :cake:

**Pull request merged**